robot/EDA/resources/EDA.robot

```diff
@@ -94,3 +94,12 @@ Create Organization Foundation
     ${account_id} =            Get Current Record Id
     Store Session Record       Account  ${account_id}
     [return]                   ${account_id}
+
+Get Records Count
+    [Documentation]         Returns the no of record identified by the given field_name and
+    ...                     field_value input for a specific object
+    [Arguments]             ${obj_name}             ${field_name}       ${field_value}
```

Comment:
Instead of ${field_name} and ${field_value}, it is better to have &{fields}. So this keyword can be used to filter multiple WHERE conditions when required.
Anyway, you don't have to modify it now if it is not required, but just a suggestion for the future.

Reply:
Sure and noted. Thank you!